services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -232,7 +232,7 @@ async def retrieve_and_notify_project_locked_state(
 
 
 @contextlib.asynccontextmanager
-async def lock_with_notification(
+async def lock_project_and_notify_state_update(
```

Comment:
I would not think I would see yo doing that 🤣

Reply:
somebody is being a bad influence ;-)